src/admin/css/wordlift-admin.css

```diff
@@ -122,3 +122,50 @@
 #wl-schemaorg-property.postbox, #wl_entity_typediv.postbox {
     border-left: 2px solid #007AFF;
 }
+
```

Comment:
@navdeepsingh can we take the chance and use SCSS instead?

Reply:
In grunt task runner, there is no task setup to parse scss files.  Need to setup and test it before implementing it.
So leaving for now.